site/content/contribute/_index.md

```diff
@@ -1,11 +1,23 @@
 ---
+title: "The contributor's guide"
+heading: "The contributor's guide"
+description: ""
+date: "2022-10-01T12:00:00-00:00"
+weight: 1
 ---
-<!DOCTYPE html>
-<html>
-    <head>
-        <title>Contribute</title>
-        <meta http-equiv="refresh" content="0; url=/contribute/getting-started" />
-    </head>
-    <body>
-    </body>
-</html>
+
+This guide is the overarching reference for all things contribution at Mattermost. Here is a brief recap to help you find what you're looking for:
+
+- Section 1: Why you might contribute
```

Comment:
Proposal to add links to sections:
[Section 1: Why you might contribute]({{< ref "/why-contribute/" >}})
[Section 2: Making good decisions]({{< ref "/good-decisions/" >}})
[Section 3: Contributor expectations]({{< ref "/expectations/" >}})
[Section 4: Where to find more information]({{< ref "/more-info/" >}})

Reply:
@neflyte - Once you have access, would you be open to helping implement Jordana's suggestion?